ci/test/pipeline.yml

```diff
@@ -113,6 +113,17 @@ steps:
           run: billing-demo
     if: $CHANGED_RUST
 
+  - id: metabase-demo
+    label: "metabase-demo"
+    depends_on: build
+    timeout_in_minutes: 5
```

Comment:
I've been using 10 in places, just to be safe. CI can be surprisingly slow. Better to burn some CPU cycles than developer cycles with spurious failures.